### PR TITLE
Assessment report page improvements

### PR DESCRIPTION
**Pull request by moiznalwalla**
https://github.com/EnterpriseDB/docs/issues/3164

### CODE REVIEW

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx [hunk 1]
```diff
@@ -109,15 +109,17 @@ An object is compatible if it successfully executes on a target database. In the
...
 | Compatible features                                            | -                                                                                                                                                               |
 | Features with existing available workaround                    | Exists                                                                                                                                                          |
 | Features with reasonable effort new workaround                 | Possible                                                                                                                                                        |
-| Features with no workaround or requiring further investigation | None / NeedsInvestigation. For features with no workaround, it might be possible to address the feature incompatibility through application-level changes. |
+| Features with no workaround or requiring further investigation | None / Needs Investigation /* |
 
+ /* For features with no workaround, it might be possible to address the feature incompatibility through application-level changes.
```
Comment (mlewandowski-edb) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx:121]: Will the /* render as * or as /*?

**Review by mlewandowski-edb — COMMENTED** [review]
See comment regarding note after table describing Feature compatibility.

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx (hunk 1 above)
Comment (mlewandowski-edb) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx:121]: Based on the deployment that @drothery-edb just generated, "/*" is being rendered as is (i.e., "/*").  Was this meant to render as "*" instead?

Comment (drothery-edb) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx:121]: @moiznalwalla to escape the * character, you need to use the backslash (\) not the forward slash (/).

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**